rules/generic/tests/Rector/ClassMethod/ChangeMethodVisibilityRector/Fixture/fixture2.php.inc

```diff
@@ -17,7 +17,7 @@ use Rector\Generic\Tests\Rector\ClassMethod\ChangeMethodVisibilityRector\Source\
 
 class ClassThatInherits2 extends ParentObject
 {
-    public function toBePublicStaticMethod() {
+    public static function toBePublicStaticMethod() {
```

Comment:
it change `ChangeMethodVisibilityRector` tests as `ParentObject::toBePublicStaticMethod()` is static too.